pkg/landscaper/execution/reconcile.go

```diff
@@ -200,7 +201,7 @@ func (o *Operation) deployOrTrigger(ctx context.Context, item executionItem) err
 	o.exec.Status.DeployItemReferences = lsv1alpha1helper.SetVersionedNamedObjectReference(o.exec.Status.DeployItemReferences, ref)
 	o.exec.Status.ExecutionGenerations = setExecutionGeneration(o.exec.Status.ExecutionGenerations, item.Info.Name, o.exec.Generation)
 	o.exec.Status.Phase = lsv1alpha1.ExecutionPhaseProgressing
-	if err := o.Client().Status().Patch(ctx, o.exec, client.MergeFrom(old)); err != nil {
+	if err := read_write_layer.PatchExecution(ctx, o.Client(), o.exec, old); err != nil {
```

Comment:
The **status** of the execution must be patched.

Reply:
yes that's another good finding